_locales/fr/messages.json

```diff
@@ -109,21 +109,158 @@
+    },
+    "showInfoButton": {
+        "message": "Montrer le bouton Info sur le lecteur YouTube"
+    },
+    "hideInfoButton": {
+        "message": "Cacher le bouton Info sur le lecteur YouTube"
+    },
+    "whatInfoButton": {
+        "message": "Il s'agit du bouton qui ouvre l'encart sur la page YouTube."
+    },
+    "hideDeleteButton": {
+        "message": "Cacher le bouton Supprimer sur le lecteur YouTube"
+    },
+    "showDeleteButton": {
+        "message": "Montrer le bouton Supprimer sur le lecteur YouTube"
+    },
+    "whatDeleteButton": {
+        "message": "Il s'agit du bouton qui permet de supprimer tous les segments commerciaux depuis le lecteur YouTube."
+    },
+    "disableViewTracking": {
+        "message": "Désactiver le suivi des vues de segments commerciaux"
+    },
+    "enableViewTracking": {
+        "message": "Activer le suivi des vues de segments commerciaux"
+    },
+    "whatViewTracking": {
+        "message": "Cette fonctionnalité suit quels segments commerciaux vous avez sautés afin de calculer le bénéfice des soumissions des \nautres utilisateurs. Elle est également utilisée comme métrique, avec les haut-votes, afin de s'assurer que les spams \nsont ignorés. L'extension envoie un message au serveur à chauqe fois qu'un segment commercial est sauté. Avec un peu de chance, peu de personnes désactiveront cette fonctionnalité afin d'obtenir des nombres de vues exacts. :)"
+    },
+    "showNotice": {
+        "message": "Afficher la notification"
+    },
+    "longDescription": {
+        "message": "SponsorBlock est une extension qui permet de passer les messages commerciaux des vidéos YouTube. SponsorBlock est une extension pour navigateur basée sur le crowdsourcing permettant à n'importe qui de soumettre le début et la fin des segments commerciaux sur les vidéos YouTube. Dès qu'une personne a soumis ces informations, les autres utilisateurs de l'extension en bénéficieront et verront les messages commerciaux automatiquement sautés.",
+        "description": "Full description of the extension on the store pages."
+    },
+    "website": {
+        "message": "Site web",
+        "description": "Used on Firefox Store Page"
+    },
+    "sourceCode": {
+        "message": "Code source",
+        "description": "Used on Firefox Store Page"
+    },
+    "noticeUpdate": {
+        "message": "La notification a été mise à jour !",
+        "description": "The first line of the message displayed after the notice was upgraded."
+    },
+    "noticeUpdate2": {
+        "message": "Si elle ne vous plaît pas, cliquez sur le bouton \"Ne plus montrer\"",
+        "description": "The second line of the message displayed after the notice was upgraded."
+    },
+    "setStartSponsorShortcut": {
+        "message": "Changer le raccourci pour enregistrer un segment"
+    },
+    "setSubmitKeybind": {
+        "message": "Changer le raccourci pour soumettre les segments"
```

Comment:
Is `raccourci` the best word to use here? Maybe `clé`?

Reply:
No `key` is a false friend in French.

* Keyboard = clavier
* Key (from keyboard) = touche
* _Key (to open a door) = clé_
* Keyboard shortcut(s) = touche(s) de raccourci / raccourci(s) clavier

And like in English you often hear "shortcut" only (not keyboard shortcut), we usually say only "raccourci" when the context is enough to understand that we're talking about the keyboard.

Reply:
Okay, thanks for the explanation. I had not heard that term before.